07-space.Rmd

```diff
@@ -5,7 +5,7 @@ Mapping is a hugely useful and powerful aspect of R and has many applications in
 This aspect of R is covered in detail in the book [Geocomputation With R](https://geocompr.robinlovelace.net/index.html) [@lovelace_geocomputation_2019].
 By mapping collision data in R, you can add layers containing other geographic datasets to further understand the reasons for certain trends. 
 This can lead to new opportunities for intervention and collaboration with other parties who may have mutually compatible solutions for reaching their goals.
-We will use the following packages in this chapter:
+We will use the following packages in this section:
```

Comment:
In the end do we not want it to be a book with chapters? Happy to switch to sections for now.

Reply:
I'm not sure, maybe something to ask Ivo about as well. I noticed that she has been changing everything to section rather than chapter.